src/app/HorizontalNav/HorizontalNav.tsx

```diff
@@ -1,72 +1,169 @@
-import React from 'react';
+import React, { useState, useEffect } from "react";
 import {
-  Nav, TextContent,Avatar,
-  Text, NavExpandable, TextVariants,NavItem, NavItemSeparator, NavList, NavGroup, PageHeader
-} from '@patternfly/react-core';
-const avatarImg = "https://www.patternfly.org/v4/images/avatarImg.668560cdf25a4932ef9f711b4acad52d.svg";
-
+  Popover,
+  Card,
+  CardBody,
+  Nav,
+  ProgressStepper,
+  TextContent,
+  Avatar,
+  Text,
+  ProgressStep,
+  TextVariants,
+  NavItem,
+  NavList,
+} from "@patternfly/react-core";
+import { get_ip, get_port } from "env";
 
 const HorizontalNav: React.FunctionComponent = () => {
+  const [activeItem, setActiveItem] = useState(0);
+  const [avtarImg, setAvtarImg] = useState("");
```

Comment:
As discussed please add a comment for all the variables which will be moved to use react context later.

Reply:
I am not very sure about what all variables need to shifted from `useState` to context api. Will take some time to think upon and get clarity and update it. [reference](https://devtrium.com/posts/how-use-react-context-pro#only-use-react-context-if-you-really-need-it)